HandPuzzle/Handpuzzelcode/Handpuzzelcode.ino

```diff
@@ -0,0 +1,23 @@
+int analogPin = A3; // potentiometer wiper (middle terminal) connected to analog pin 3
+                    // outside leads to ground and +5V
+int val = 0;  // variable to store the value read
+
+void setup() {
+  Serial.begin(9600); //  setup serial
+  pinMode(13, OUTPUT); //led
+  pinMode(12, OUTPUT); //solenoid
+}
+
+void loop() {
+  val = analogRead(analogPin);  // read the input pin
+  Serial.println(val);          // debug value
+  if (val < 700) {
```

Comment:
700 is weer een magic number. Waarom is het 700 of uberhaupt, wat stelt die 700 voor?